src/Models/EventModel.php

```diff
@@ -189,10 +197,10 @@ public function getEventsByCategoryId(string $categoryId, int $previousMonths =
     public function getUpcomingEventsByCategoryIdAndSearch(?string $categoryId = null, ?string $searchQuery = ''): array
     {
         $sql = 'SELECT `events`.`id`, `events`.`name`, `events`.`category`, 
-        `event_categories`.`name` AS `category_name`, `location`, `date`, `start_time`,`end_time`, `notes`
+        `event_categories`.`name` AS `category_name`, `location`, `date`, `start_time`,`end_time`, `notes`, `deleted`
```

Comment:
I don't think you need to select `deleted` here, it's just enough to have `WHERE deleted = 0` later